src/test/scala/is/hail/variant/IntervalSuite.scala

```diff
@@ -152,7 +152,7 @@ class IntervalSuite extends SparkSuite {
   }
 
   @Test def testFilter() {
-    val vds = hc.importVCF("src/test/resources/sample2.vcf", nPartitions = Some(4)).cache()
+    val vds = hc.importVCF("src/test/resources/sample2.vcf", nPartitions = Some(4)).persist()
```

Comment:
Is this to fix the slow test?

Reply:
Oh, oops. I didn't mean to leave this in.

Reply:
changing back now.

Reply:
done!